src/SprykerEco/Shared/ComputopApi/Transfer/computop_api.transfer.xml

```diff
@@ -132,8 +137,7 @@
         <property name="payID" type="string"/>
         <property name="xID" type="string"/>
         <property name="transID" type="string"/>
-        <property name="orderid" type="string"/>
-        <property name="paypalurl" type="string"/>
+        <property name="orderId" type="string"/>
```

Comment:
please revert transfer property deleting since it's bc-breaking

Reply:
Fixed

Reply:
case is different from original
please revert

Reply:
Fixed

Reply:
please check cases with incorrect case usage in code for `<property name="orderid" type="string"/>`
I see at least `FacadeTestHelper:362` is using wrong case

Reply:
Can you please specify what you mean with _using wrong case_? It's calling the generate method of ComputopApiPayPalExpressPrepareResponseTransfer::setOrderId() and is imho fine.

Reply:
Fixed